communication/contributor-comms/storytelling-resources/README.md

```diff
@@ -0,0 +1,35 @@
+# Storytelling resources
+
+We have collected some information to get anyone started:
```

Comment:
Add a little bit to this affect: "If you're looking to get started blogging on the K8s.dev blog this page contains a list of resources available to help you be successful."